--- /tmp/original_idracclient.py	2026-01-12 23:47:59.317396730 -0600
+++ idracclient.py	2026-01-12 23:51:33.088638927 -0600
@@ -61,8 +61,7 @@
 
     args = cmd.parse_args()
 
-    loop = asyncio.get_event_loop()
-    ret = loop.run_until_complete(run(args))
+    ret = asyncio.run(run(args))
 
     exit(ret)
 
@@ -213,6 +212,7 @@
     invocation = [
         args.java,
         "-cp", str(mainjar),
+        "--enable-native-access=ALL-UNNAMED",
     ]
 
     if not args.no_native_libs:
@@ -231,6 +231,7 @@
         "version=2",
         "reconnect=2",
         "vmprivilege=true",
+        "vm=1",
         "helpurl=https://%s:%d/help/contents.html" % (java_kvmhost, java_idracport),
     ])
 
